File: evenodd.js
const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
});

readline.question('Enter a number: ', (num) => {
    num = Number(num);
    console.log(num % 2 === 0 ? "Even" : "Odd");
    readline.close();
});

File: evenodd.go
package main

import ("fmt")

func EvenOdd(num int) string {
    results := [2]string{"Even", "Odd"}  
    return results[num%2]
}

func main() {
    var n int
    fmt.Print("Enter a number: ")
    fmt.Scan(&n)
    fmt.Println(EvenOdd(n))
}